BUILDING.md

```diff
@@ -0,0 +1,163 @@
+- **When to use**: When you’re ready to share the ontology with others.
+- **Output**: The ZIP file appears in the project root.
+
+### Fix Sources Early
+
+```bash
+mvn -Pfix install
+```
+
+- **What it does**: Runs all source-modifying tasks (inference and formatting) right at the start, then completes the full build.
+- **When to use**: If you’ve added new units or prefixes and need to update `VOCAB_QUDT-UNITS-ALL.ttl` before validation.
+
+## Special Commands for RDF Tasks
+
+These commands are for specific ontology tasks—like inferring new triples—and aren’t part of the automatic build. They’re perfect for tweaking the RDF data directly. Run them as needed.
+1. **Update All Source Inferences and Formatting**
+
+```bash
+mvn seq:run@infer-and-format
+```
+
+- **What it does**: Infers `qudt:hasFactorUnit` and `qudt:scalingOf` triples, merges them into `src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl`, and formats the result.
+- **When to use**: To refresh all inferred data in the source file at once e.g., after adding new units or prefixes.
+
+2. **Infer and Merge Factor Units**
+
+   ```bash
+   mvn seq:run@infer-factorUnits
+   ```
+
+   - **What it does**: Infers `qudt:hasFactorUnit` triples e.g., relationships between units like "meter per second", merges them into `VOCAB_QUDT-UNITS-ALL.ttl`, and formats it.
+   - **When to use**: If validation fails and says factor units are missing or wrong (check `target/validation/validationReportSrc-factorUnits.ttl`).
+3. **Infer and Merge Scaling Relationships**
+
+   ```bash
+   mvn seq:run@infer-scalingOf
+   ```
+
+   - **What it does**: Infers `qudt:scalingOf` triples e.g., how units scale with prefixes like "kilo", merges them into `VOCAB_QUDT-UNITS-ALL.ttl`, and formats it.
+   - **When to use**: If validation flags scaling issues (check `target/validation/validationReportSrc-factorUnits.ttl`).
+
+## How It Works: Key Steps
+
+The build process is split into stages (Maven calls them "phases"). Here’s what happens, in terms an RDF expert might appreciate:
+- **Validate**: Sets up properties like the version `3.0.1-SNAPSHOT` and prepares SHACL templates from `src/build/srcgen/`.
+- **Process Sources**: Checks that all `.ttl` files in `src/` are formatted and valid against SHACL shapes (e.g., `QUDT_SRC_QA_TESTS.ttl`).
+- **Compile**: Infers triples like `qudt:applicableUnit` and merges them into `target/dist/` files.
+- **Process Resources**: Adds links to IEC standards e.g., `qudt:informativeReference` for units.
+- **Test**: Validates the final output in `target/dist/` against stricter SHACL tests.
+- **Install** (with `-Pzip`): Optionally creates the ZIP.
+The special commands above like `mvn seq:run@infer-factorUnits` let you run inference and merging steps manually, updating the source files directly instead of waiting for the full build.
+
+## Troubleshooting
+
+- **TL;DR**: If you don't know or don't want to dig deep, run:
```

Comment:
What does TL;DR mean?

Reply:
"Too long; didn't read". Often used to mark the abstract of a blog post.